geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/ShowMetricsCommand.java

```diff
@@ -201,15 +198,16 @@ private ResultData getMemberMetrics(DistributedMember distributedMember,
         ErrorResultData erd = ResultBuilder.createErrorResultData();
```

Comment:
We do not need to use the ResultBuilder anymore. The result model error is fine.